sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/UnsafeArraySuite.scala

```diff
@@ -204,4 +204,29 @@ class UnsafeArraySuite extends SparkFunSuite {
     val doubleEncoder = ExpressionEncoder[Array[Double]].resolveAndBind()
     assert(doubleEncoder.toRow(doubleArray).getArray(0).toDoubleArray.sameElements(doubleArray))
   }
+
+  test("from binary array") {
+    val blobs = Array[Array[Byte]](
```

Comment:
It might be useful to declare an array of ints, and map them to bytes, rather than write `.toByte` a bunch of times. Maybe a private utility method would make it worthwhile.